CIME/scripts/create_newcase.py

```diff
@@ -231,12 +231,14 @@ def parse_command_line(args, cimeroot, description):
         "--input-dir",
         help="Use a non-default location for input files. This will change the xml value of DIN_LOC_ROOT.",
     )
-    parser.add_argument(
-        "--driver",
-        default=get_cime_default_driver(),
-        choices=("mct", "nuopc", "moab"),
-        help=argparse.SUPPRESS,
-    )
+    if model == "cesm":
+        parser.add_argument(
+            "--driver",
+            default=get_cime_default_driver(),
+            choices=("mct", "nuopc", "moab"),
```

Comment:
moab is not supported by cesm, you should remove that choice.

Reply:
OK, I'll do that.